setup.md

```diff
@@ -4,24 +4,24 @@ title: Setup
 ---
 
 To participate in this Library Carpentry lesson, you will need a working UNIX-like shell environment.
-Specifically we will be using Bash ([Bourne Again Shell](https://en.wikipedia.org/wiki/Bash_(Unix_shell))) which is standard on Linux and Mac OS. 
-Even if you are a Windows user, learning Bash will open up a powerful set of tools on your personal machine, in addition to familiarizing you with the standard remote interface used on almost all servers and super computers. 
+Specifically we will be using Bash ([Bourne Again Shell](https://en.wikipedia.org/wiki/Bash_(Unix_shell))) which is standard on Linux and macOS.
+Even if you are a Windows user, learning Bash will open up a powerful set of tools on your personal machine, in addition to familiarizing you with the standard remote interface used on almost all servers and super computers.
 
 >## Terminal Setup
 >
->Bash is the default shell on most Linux distributions and Mac OS. 
+>Bash is the default shell on most Linux distributions and Mac OS.
```

Comment:
@katrinleinweber Still needs to be changed to macOS. I guess this means we will have to make this change throughout the lessons.